samples/v1.2/Scenarios/ExpenseReport.json

```diff
@@ -198,54 +198,36 @@
 					"width": "auto"
 				},
 				{
-					"type": "Column",
-					"id": "chevronDown1",
-					"spacing": "Small",
-					"verticalContentAlignment": "Center",
-					"items": [
-						{
-							"type": "Image",
-							"selectAction": {
-								"type": "Action.ToggleVisibility",
-								"title": "collapse",
-								"targetElements": [
-									"cardContent1",
-									"chevronUp1",
-									"chevronDown1"
-								]
-							},
-							"url": "https://adaptivecards.io/content/down.png",
-							"width": "20px",
-							"altText": "Air travel expense collapsed"
-						}
-					],
-					"width": "auto"
-				},
-				{
-					"type": "Column",
-					"id": "chevronUp1",
-					"isVisible": false,
-					"spacing": "Small",
-					"verticalContentAlignment": "Center",
-					"items": [
-						{
-							"type": "Image",
-							"selectAction": {
-								"type": "Action.ToggleVisibility",
-								"title": "expand",
-								"targetElements": [
-									"cardContent1",
-									"chevronUp1",
-									"chevronDown1"
-								]
-							},
-							"url": "https://adaptivecards.io/content/up.png",
-							"width": "20px",
-							"altText": "Air travel expense expanded"
-						}
-					],
-					"width": "auto"
-				}
+                    "type": "Column",
```

Comment:
looks like the whitespace here (and below) doesn't line up with the rest of the file -- you should consider [configuring your editor](https://editorconfig.org/#download) to consume our `.editorconfig`

Reply:
Fixed, that spec is cool - thanks!